tool/cli/cmd/grpcmgr/hadbmgr.go

```diff
@@ -256,3 +257,42 @@ func GetHaDbInfo() (string, string, string, string, string, error) {
 
 	return host, port, username, password, dbname, nil
 }
+
+func GetIpv4(nodeName string) (string, error) {
+	db, err := OpenHaDb()
+	if err != nil {
+		return "", errors.New("an error occured while opening ha database: " + err.Error())
+	}
+
+	// close database
+	defer db.Close()
+	// check db
+	err = db.Ping()
```

Comment:
Yes, we can check in either ways. However, I believe this is a little bit better. By executing Ping() before executing the actual query, we can make it sure if the error is because of database or query.

Reply:
Ok. Got it